app/views/email_signups/new.html.erb

```diff
@@ -1,14 +1,29 @@
-<% page_title "Email alert subscription" %>
+<% page_title t("email_signups.new.title") %>
 <% page_class "email-signup-show" %>
 
 <header class="block email-header">
   <div class="inner-block">
-    <h1>Email alert subscription</h1>
+    <h1 class="govuk-heading-l"><%= t("email_signups.new.title") %></h1>
   </div>
 </header>
 
 <div class="block">
   <div class="inner-block">
-    <%= render 'form', email_signup: @email_signup %>
+    <p class="govuk-body">
+      <%= t("email_signups.new.description") %>
+    </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-8">
+      <strong style="font-weight: bold"><%= @email_signup.name %></strong>
```

Comment:
Similar concern to https://github.com/alphagov/finder-frontend/pull/2315#pullrequestreview-537788983 regarding the inline style.

Reply:
Yup - same problem too! I've updated the commit message to explain.

Reply:
And the same solution ;p